Rewrite simply "Localhost free term" in README.md within #10510 (#12415)
tauri-apps/tauri: README.md

- Built-in self updater (desktop only)
- System tray icons
- Native notifications

> - Native Webview API (tauri doesn't create http(s) server for localhost to provide contents for webview.

Reviewer: ```suggestion
- Native WebView Protocol (tauri doesn't create a localhost http(s) server to serve the WebView contents.
```

> [some webview-based ui framework depends on localhost http(s) sever launched internally, but tauri doesnt.])

Reviewer: maybe we can remove this bit
